.github/workflows/main.yml

```diff
@@ -28,16 +28,6 @@ jobs:
             access: row_major
```

Comment:
@strasdat please note that row_major / col_major is broken, as Eigen is installed as part of the super_project_build, and this needs to be plumbed into that project.

Reply:
It seems like the env variable is set correctly here. Let me check...

Reply:
okay, this is not set correctly in the main.yml. Turning it off for now...

Reply:
@ethanrublee, I made a merge mistake. Will re-enable your changes... about row-major.